### ISSUE TITLE
latest versions of Quokka extension for VS Code seem broken

### ISSUE THREAD

**Issue by marcelh-gh** [wallabyjs/quokka#242]
I am using Quokka for several projects. All of them worked until just a few days ago.
Now Quokkka won't run on any of them.
It gives all kinds of weird errors about typescript compiler options.
The tsconfig files are not changed.

Here is an example error:
```
Quokka #1 (node: v10.5.0, TypeScript: v3.0.1)​​​​
​​​​
​​⨯ Unable to compile TypeScript:​​
​​error TS5023: Unknown compiler option 'configFilePath'.
​​
​​​​
  at ​​​createTSError​​​ ​./node_modules/ts-node/src/index.ts:253​
  at ​​​Object.register​​​ ​./node_modules/ts-node/src/index.ts:257​
```

Visual Studio Code v1.26.1
Windows 10 1709

I have reverted the extension to version 1.0.138 or 1.0140 - the errors don't show up but Quokka still does not do anything other than showing
`​​​​Quokka #1 (node: v10.5.0, TypeScript: v3.0.1)​​​​`

**Comment by marcelh-gh** [comment]
btw is there a changelog for the extension or Quokka in general? I have not found one. It would be great to be able to see that. Thanks a lot!

**Comment by smcenlly** [comment]
We made a change last week for Quokka to read tsconfig files the same way that TypeScript's "tsc" compiler does (previously they were ignored). 

Could you please send us a copy of your project structure and tsconfig.json files and explain which files in which folders are not working properly?

**Comment by marcelh-gh** [comment]
What about a changelog? What's the last version before that change? 
I need to get up and running again.
Thanks!

BTW: wallaby works fine. Both wallaby and quokka are pro version.

**Comment by marcelh-gh** [comment]
Multiple projects, different configs - simple and and complex. All don't work anymore. Error messages don't make sense, for example "value for target has to be a string", where target is set to "ES2018"...

Please do some testing with any kind of tsconfig....

**Comment by marcelh-gh** [comment]
About that error in my OP: a compiler option `configFilePath` neither exists nor is it used in my tsconfig

**Comment by smcenlly** [comment]
Thanks for the additional details. These changes are only in Quokka (not Wallaby). The version prior to these changes is Quokka version v1.0.136. We've duplicated the issue and are investigating the root cause.

Currently, we typically update our change.log on a quarterly basis. We are in the process of automating updates so that they occur more frequently.

**Comment by marcelh-gh** [comment]
Thank you for the quick reply. :-)

**Comment by ArtemGovorov** [comment]
The issue is fixed and the fix is published in v1.0.142.
